privaterelay/templates/includes/premium-onboarding.html

```diff
@@ -0,0 +1,173 @@
+                                    minlength="1"
+                                    maxlength="63"
+                                    >
+                                    <input class="mzp-c-button mzp-t-product" type="submit" value="{% ftlmsg 'banner-register-subdomain-button-search' %}">
+                                </form>
+                            </div>
+                        {% endif %}
+                        
+                        <div class="c-premium-onboarding-action-completed {% if user_profile.subdomain %} is-visible {% endif %}">
+                            <img class="margin-right" src="/static/images/icon-check.svg" alt="">
+                            <samp class="js-premium-onboarding-domain-registration-preview">{{ user_profile.custom_domain }}</samp>
+                        </div>
+                    </div>
+                </div>
+                
+            </div>
+        </li>
+        <!-- Step 3 -->
+        <li class="c-premium-onboarding-step c-premium-onboarding-step-3 {% if user_profile.onboarding_state == 2 %} is-visible {% endif %}">
+            <div class="mzp-l-content mzp-t-content-xl">
+                <div class="c-premium-onboarding-headlines">
+                    <h1>Block, Forward, and Reply!</h1>
+                    <h2>Now you can control what hits your inbox, one email at a time.</h2>
+                </div>
+
+                <div class="c-premium-onboarding-content">
+                    <div class="c-premium-onboarding-content-image">
+                        <img src="{% static 'images/dashboard-onboarding/woman-email.svg' %}" alt="Woman next to large envelope">
+                    </div>
+                    <div class="c-premium-onboarding-content-description">
+                        <p><strong>Reply to emails without giving away your real address:</strong> Need to reply to emails sent to an alias? Just reply as normal—your alias will still protect your email address.</p>
+                        <p><strong>Get the Relay extension for Firefox</strong> The Relay extension for Firefox makes using email aliases even easier. Reply to emails without giving away your real address: Need to reply to emails sent to an alias? Just reply as normal—your alias will still protect your email address.</p>
+                        <a target="_blank" href="https://addons.mozilla.org/firefox/addon/private-relay/?utm_source=fx-relay&utm_medium=onboarding&utm_campaign=install-addon" class="c-premium-onboarding-install-addon mzp-c-button mzp-t-product">Get Relay extension</a>
+                        <div class="c-premium-onboarding-action-completed">
+                            <img class="margin-right" src="/static/images/icon-check.svg" alt="">
+                            Relay extension installed!
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </li>
+    </ol>
+    <div class="c-premium-onboarding-progress">
```

Comment:
Because of the design (and not wanting to get into a `position:absolute` muck), the action buttons for each step are stored apart from the content of each step. You'll see in the JS how the `is-visible` class gets walked through each list to show the correct step at the right time.